homeassistant/components/network/network.py

```diff
@@ -0,0 +1,79 @@
+"""Network helper class for the network integration."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from homeassistant.core import HomeAssistant, callback
+
+from .const import (
+    ATTR_CONFIGURED_ADAPTERS,
+    DEFAULT_CONFIGURED_ADAPTERS,
+    NETWORK_CONFIG_SCHEMA,
+    STORAGE_KEY,
+    STORAGE_VERSION,
+)
+from .models import Adapter
+from .util import (
+    adapters_with_exernal_addresses,
+    async_load_adapters,
+    enable_adapters,
+    enable_auto_detected_adapters,
+)
+
+ZC_CONF_DEFAULT_INTERFACE = (
+    "default_interface"  # cannot import from zeroconf due to circular dep
```

Comment:
Alternative is to import it inside the function where it's used.

Reply:
switched to the late import